src/store/state.js

```diff
@@ -1,6 +1,43 @@
 'use strict'
 
 export default {
-  noteList: [],
-  curriculumList: []
+  noteList: [
+    { title: 'カリキュラムタイトル', thumbUrl: './static/img/student.jpg', memo: 'フリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキスト', date: '2018.02.10' },
+    { title: 'カリキュラムタイトル', thumbUrl: './static/img/student.jpg', memo: 'フリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキスト', date: '2018.02.10' },
+    { title: 'カリキュラムタイトル', thumbUrl: './static/img/student.jpg', memo: 'フリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキスト', date: '2018.02.10' },
+    { title: 'カリキュラムタイトル', thumbUrl: './static/img/student.jpg', memo: 'フリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキストフリーフォーマットでのテキスト', date: '2018.02.10' },
+
+  ],
+  curriculumList: [
```

Comment:
Should I change the curriculum list screen to get data from the state as well?

https://github.com/dank1ntq/app/blob/1d44c004228c8159813aa1c11832527f7903f575/src/components/CurriculumList.vue#L54-L75

Reply:
You should when you can do it

Reply:
I got it.
I will fix it with another PR. I have no time to say that ...